src/utils.c

```diff
@@ -323,7 +323,7 @@ str_replace_all(char **const text, const char *templt, const char *str)
 	char *occ;
 
 	if (str_len <= templt_len) {
-		char *new_str = malloc(templt_len);
+		char *new_str = malloc(templt_len + 1);
```

Comment:
What is the warning/error you are fixing here?

Reply:
```
CID 180942 (#1 of 1): Out-of-bounds access (OVERRUN)
3. alloc_strlen: Allocating insufficient memory for the terminating null of the string.
```